lib/utils/getDemoPoolContractAddress.js

```diff
@@ -1,11 +1,8 @@
 import { CONTRACT_ADDRESSES } from 'lib/constants'
 
 export const getDemoPoolContractAddress = (networkName, ticker) => {
-  let chainId = 1
-  if (networkName === 'kovan') {
-    chainId = 42
-  }
-
+  const chainNameToId = {mainnet: 1, ropsten: 3, rinkeby: 4, kovan: 42, buidlerEVM: 31337}
```

Comment:
There are two `utils` files, one is `nameToChainId.js` and the other is `chainIdToName.js`. Ideally we'd import and use those instead of defining this here.

Reply:
fixed!